src/main/java/io/dmuncle/maven/DmUncleMavenMojo.java

```diff
@@ -42,19 +44,17 @@
     private JSONArray systemDeps;
 
     public void execute() throws MojoExecutionException, MojoFailureException {
+        cleanUp();
```

Comment:
this cleanup will later be done with a seperate task that is called on mvn clean